devices/xiaomi.js

```diff
@@ -2282,16 +2383,28 @@ module.exports = [
         description: 'Aqara Smart Radiator Thermostat E1',
         fromZigbee: [fzLocal.aqara_trv, fz.thermostat, fz.battery],
         toZigbee: [tzLocal.aqara_trv, tz.thermostat_occupied_heating_setpoint],
-        exposes: [e.switch().setAccess('state', ea.STATE_SET),
-            exposes.climate().withSetpoint('occupied_heating_setpoint', 5, 30, 0.5)
-                .withLocalTemperature(ea.STATE).withPreset(['manual', 'away', 'auto'], ea.STATE_SET),
-            e.child_lock(), e.window_detection(), e.valve_detection(),
-            e.away_preset_temperature(), e.battery_voltage(), e.battery(),
+        exposes: [
+            exposes.switch()
+                .withState('state', true, 'Turn off the thermostat', ea.ALL, 'OFF', 'ON'),
+            exposes.climate()
+                .withSetpoint('occupied_heating_setpoint', 5, 30, 0.5)
+                .withLocalTemperature(ea.STATE)
+                .withPreset(['manual', 'away', 'auto']).setAccess('preset', ea.ALL),
+            e.temperature_sensor_select(['internal', 'external']).withAccess(ea.ALL),
+            exposes.binary('calibrated', ea.STATE, true, false)
+                .withDescription('Is the valve calibrated'),
```

Comment:
Can you explain a bit more what this does?

Reply:
"calibrated": the sign of valve calibration has the values 1 - the valve is calibrated and 0 - the valve is not calibrated
"sensor": internal - internal temperature sensor is used; external - an external temperature sensor is used

Reply:
But was does "valve calibrated" mean?

Reply:
Valve finds extreme positions, first it closes completely, and then it opens completely, after that it is considered that the thermal head can control the temperature